src/commands/kv/namespace/site.rs

```diff
@@ -0,0 +1,60 @@
+use cloudflare::endpoints::workerskv::create_namespace::CreateNamespace;
+use cloudflare::endpoints::workerskv::create_namespace::CreateNamespaceParams;
+use cloudflare::endpoints::workerskv::list_namespaces::ListNamespaces;
+use cloudflare::endpoints::workerskv::WorkersKvNamespace;
+use cloudflare::framework::apiclient::ApiClient;
+use cloudflare::framework::response::ApiFailure;
+
+use crate::commands::kv;
+use crate::settings::global_user::GlobalUser;
+use crate::settings::target::Target;
+use crate::terminal::message;
+
+pub fn site(target: &Target, user: &GlobalUser) -> Result<WorkersKvNamespace, failure::Error> {
+    let client = kv::api_client(user.to_owned())?;
+
+    let title = format!("__{}-{}", target.name, "workers_sites_assets");
+    let msg = format!("Creating namespace for Workers Site \"{}\"", title);
+    message::working(&msg);
+
+    let response = client.request(&CreateNamespace {
```

Comment:
Couldn't we use `kv::namespace::create::create()?` I think this would reduce a lot of redundancy.

Reply:
i'd be stoked to make refactors later.